docs/reference/rollup/apis/stop-job.asciidoc

```diff
@@ -22,6 +22,14 @@ Stopping an already stopped job has no action.
 `job_id` (required)::
   (string) Identifier for the job
 
+==== Query Parameters
+
+`wait_for_stopped` (optional)::
```

Comment:
`wait_for_completion` for consistency with the other APIs that waits for the completion of the command ?

Reply:
Also it might be difficult for a user to set up a value for the timeout. So maybe we should make it a `boolean` and have another parameter (`timeout`)  that could default to `30s` ?

Reply:
Yeah, I was hoping to avoid two parameters, since configuring a timeout without setting `wait_for_completion` to true won't do anything.  But I can see how it's a strange setup.  I could go either way

Reply:
The two params solution is consistent with the GetTask API but I don't have a strong opinion here either.